Documentation/Storage-Configuration/Block-Storage-RBD/rbd-mirroring.md

```diff
@@ -182,13 +182,13 @@ To achieve RBD Mirroring, `csi-omap-generator` and `volume-replication`
 
 * **Volume Replication Operator**: Volume Replication Operator is a
  kubernetes operator that provides common and reusable APIs for
- storage disaster recovery.
+ storage disaster recovery. The volume replication operation is
+ supported by the [CSIAddons](https://github.com/csi-addons/kubernetes-csi-addons#readme)
  It is based on [csi-addons/spec](https://github.com/csi-addons/spec)
  specification and can be used by any storage provider.
- For more details, refer to [volume replication operator](https://github.com/csi-addons/volume-replication-operator).
 
 Execute the following steps on each peer cluster to enable the
- OMap generator and Volume Replication sidecars:
+ OMap generator and CSIADDONS sidecars:
```

Comment:
CSI-Addons is the name of the project, maybe use that like you do later in this document?

Reply:
@nixpanic how about having a separate cleanup PR for it? For now, we are following this everywhere else in the repo. Changing at one place would require changes else where too.